packages/core/src/controllers/SnackController.ts

```diff
@@ -2,22 +2,35 @@ import { subscribeKey as subKey } from 'valtio/vanilla/utils'
 import { proxy } from 'valtio/vanilla'
 import { CoreHelperUtil } from '../utils/CoreHelperUtil.js'
 
+// -- Constants ----------------------------------------- //
+const DEFAULT_STATE = Object.freeze<SnackControllerState>({
```

Comment:
why freeze this?

Reply:
This is to be the reference for default state and must not change in runtime